src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfileController.php

```diff
@@ -286,6 +287,10 @@ public function bulkDeleteAction(Request $request)
     protected function getErrorMessages()
     {
         return [
+            UploadedImageConstraintException::class => $this->trans(
+                'Image format not recognized, allowed formats are: %s',
+                'Admin.Notifications.Error'
+            ),
```

Comment:
Hello @lartist  :)
Missing allowed formats printf params. You can use  `ImageManager::EXTENSIONS_SUPPORTED`.

```php
UploadedImageConstraintException::class => $this->trans(
    'Image format not recognized, allowed formats are: %s',
    'Admin.Notifications.Error',
    [
        implode(', ', ImageManager::EXTENSIONS_SUPPORTED),
    ]
),
```

See `PrestaShopBundle\Controller\Admin\ProductImageController::uploadImageAction` for example

Reply:
Nice suggestion 👍